src/nvim/ex_docmd.c

```diff
@@ -271,7 +271,7 @@ do_exmode (
 int do_cmdline_cmd(char *cmd)
 {
   return do_cmdline((char_u *)cmd, NULL, NULL,
-      DOCMD_VERBOSE|DOCMD_NOWAIT|DOCMD_KEYTYPED);
+                    DOCMD_NOWAIT|DOCMD_KEYTYPED);
```

Comment:
`DOCMD_VERBOSE` was removed in several places. This wasn't in the original PR, was it discussed somewhere, or could you explain?

Reply:
Is there a conclusion about why this was removed from `do_cmdline_cmd()`?

Reply:
I've created a PR #23482 to revert this change.